ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -117,25 +110,25 @@ private fun PurchaseButton(state: PaywallViewState.Template2, viewModel: Paywall
         verticalArrangement = Arrangement.Center,
     ) {
         val activity = LocalContext.current.getActivity() ?: error("Error finding activity")
-        val colors = state.paywallData.getColors()
+        val colors = state.templateConfiguration.getColors(isSystemInDarkTheme())
```

Comment:
Per my other comment, that way we wouldn't have to call `isSystemInDarkTheme` everywhere.